help/en/html/web/JsonServlet.shtml

```diff
@@ -193,12 +195,123 @@
           <a href="http://localhost:12080/web/power.html">Power Demonstration</a>  (with
           JMRI Web Server running).
         </dt>
-
         <dd>
           source <a href=
           "https://github.com/JMRI/JMRI/blob/master/web/power.html">/web/power.html</a>
         </dd>
-      </dl>
```

Comment:
It seems to me that this `</dl>` tag should be restored. It's currently after your additions. Maybe the current HTML works with web browsers but I'm concerned about screen readers. I know there are people who are blind who uses JMRI and they uses screen readers.

Reply:
Thanks, @danielb987 I moved my section out of the <dl>.